tests/system/providers/google/cloud/transfers/example_cloud_storage_transfer_service_gcp.py

```diff
@@ -100,12 +103,12 @@
 # [END howto_operator_gcp_transfer_update_job_body]
 
 with models.DAG(
-    "example_gcp_transfer",
+    DAG_ID,
+    schedule='@once',
     start_date=datetime(2021, 1, 1),
     catchup=False,
-    tags=["example"],
+    tags=["example", "gcp"],
 ) as dag:
-
     create_transfer = CloudDataTransferServiceCreateJobOperator(
```

Comment:
In old design, pytest class, we're creating bucket before test:
```
GoogleSystemTest.create_gcs_bucket(GCP_TRANSFER_SECOND_TARGET_BUCKET, location="asia-east1")
GoogleSystemTest.create_gcs_bucket(GCP_TRANSFER_FIRST_TARGET_BUCKET)
```

We need to create those buckets here as well (and remove them with trigger ALL_DONE) - refer to other system tests to see how it could be done.